backend/endpoints/rooms.py

```diff
@@ -1,9 +1,11 @@
 from fastapi import APIRouter, BackgroundTasks, HTTPException, status
 from fastapi.responses import JSONResponse
 from schema.response import ResponseModel
-from schema.room import Room, RoomRequest
+from schema.room import AddToRoom, Role, Room, RoomRequest, RoomType
+from starlette.responses import Response
```

Comment:
Can imports like this be added to one file so everyone can use it? That way the number of code lines will reduce.